core/src/main/scala/filodb.core/metadata/Dataset.scala

```diff
@@ -115,7 +115,7 @@ object DatasetOptions {
                                       metricColumn = "__name__",
                                       // defaults that work well for Prometheus
                                       ignoreShardKeyColumnSuffixes =
-                                        Map("__name__" -> Seq("_bucket", "_count", "_sum")),
+                                        Map("__name__" -> Seq("_bucket", "_count", "_sum", "_min", "_max")),
```

Comment:
@sandeep6189 this is not needed, we can revert this change

Reply:
Ack, reverting